### PR TITLE
Docs Python Guide

### LINKED ISSUE #1730: Add python guide section

### PR DESCRIPTION
**Pull request by abemassry**
closes #1730

### CODE REVIEW

File: docs/src/components/guides/python/Introduction.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+
+export default function Introduction() {
+  return (
+    <section className="GettingStartedPage">
+      <h1>Getting started with Linode Python</h1>
+      <section>
+        <p>
+          With the release of API 4, Linode has also released an official python library.  This guide is a simple introduction to working with
+          the python library - if you're looking for comprehensive documentation, please refer to the [Python Library Reference](/libraries/python-reference).
```
Comment (mparke) [inline docs/src/components/guides/python/Introduction.js]: library reference url here needs to be removed or updated

File: docs/src/components/guides/python/Introduction.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+          The official Linode python library is open-source on [github](http://github.com/Linode/python-linode-api), and can be installed through
+          pypi with:
+        </p>
+        <pre>
+          <code>pip install linode-api</code>
+        </pre>
+        <p>
+          In order to make requests to the API, you're going to need an OAuth Token.  If you haven't already, log in to the <a href="https://cloud.linode.com">Mangager Beta</a>, click "Manage personal access tokens" and "Generate a new token".  This will generate and display a valid OAuth Token with access to your entire (alpha) account, which we will use for this tutorial.
```
Comment (mparke) [inline docs/src/components/guides/python/Introduction.js]: manager typo

File: docs/src/components/guides/python/Introduction.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+        <p>
+          Looks like we only have one option in the alpha environment - so we’ll just grab it and move on.
+        </p>
+        <pre>
+          <code>
+          >>> dc = client.get_datacenters()[0]
+          </code>
+        </pre>
+        <p>
+          For a Service, we know that we want “Linode 2048”. Since we know the service type’s label, we don’t need to list all services, we can just use the label:
```
Comment (mparke) [inline docs/src/components/guides/python/Introduction.js]: are the double quotes different here?

File: docs/src/components/guides/python/Introduction.js [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+          Now that we have a distribution, let’s make a new Linode running it! But first, let’s clean up that first Linode that we don’t really need:
+        </p>
+        <pre>
+          <code>
+{`>>> l.delete()
+True`}
+          </code>
+        </pre>
+        <p>
+          Now that that’s gone, we can create a new Linode running Debian 8.8:
```
Comment (mparke) [inline docs/src/components/guides/python/Introduction.js]: does this debian version need to match exactly with the code example, `8.8 vs 8`?

File: docs/src/components/guides/python/Introduction.js [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+        </p>
+        <pre>
+          <code>
+            >>> l, pw = client.linode.create_instance(serv, dc, distribution="linode/debian8")
+          </code>
+        </pre>
+        <p>
+          This time, we called <code>create_instance</code> with a “distribution” keyword argument. This tells the API what we want on the new Linode, and it will do “the right thing” to give you a working configuration. Since a Distribution needs a root password and we didn’t provide one, the client helpfully generated one for us and returned it as well. Let’s boot it and wait for it to come online:
```
Comment (mparke) [inline docs/src/components/guides/python/Introduction.js]: same quotes question from above, might want to check everywhere

File: docs/src/components/guides/python/Introduction.js [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+export default function Introduction() {
+  return (
+    <section className="GettingStartedPage">
+      <h1>Getting started with Linode Python</h1>
+      <section>
+        <p>
+          With the release of API 4, Linode has also released an official python library.  This guide is a simple introduction to working with
+          the python library.
+          The official Linode python library is open-source on [github](http://github.com/Linode/python-linode-api), and can be installed through
```
Comment (eatonphil) [inline docs/src/components/guides/python/Introduction.js]: Use a Link?

File: docs/src/components/guides/python/Introduction.js [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+export default function Introduction() {
+  return (
+    <section className="GettingStartedPage">
+      <h1>Getting started with Linode Python</h1>
+      <section>
+        <p>
+          With the release of API 4, Linode has also released an official python library.  This guide is a simple introduction to working with
+          the python library.
+          The official Linode python library is open-source on <a href="http://github.com/Linode/python-linode-api">github</a>, and can be installed through
+          pypi with:
```
Comment (eatonphil) [inline docs/src/components/guides/python/Introduction.js]: Editorial comment: could we just drop the `through pypi with` part? No one cares that its hosted on the defacto hosting platform.

File: docs/src/components/guides/python/Introduction.js [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+          <h5>by William Smith; last updated May 22nd, 2017</h5>
+      </header>
+      <section>
+        <p>
+          With the release of API V4, Linode has also released an official Python library.  This guide is a simple introduction to working with
+          the Python library.
+          The official Linode Python library is open source on <a href="http://github.com/Linode/python-linode-api">Github</a>, and can be installed:
+        </p>
+        <pre>
+          <code>pip install linode-api</code>
```
Comment (eatonphil) [inline docs/src/components/guides/python/Introduction.js]: `$ pip` or no?

Comment (eatonphil) [inline docs/src/components/guides/python/Introduction.js]: @abemassry ping

File: docs/src/components/guides/python/Introduction.js [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+        </pre>
+        <p>
+          In order to make requests to the API, you're going to need an OAuth Token. You were given one when you signed up for an alpha account, if you've lost yours contact support.
+        </p>
+      </section>
+      <section>
+        <h2>Connecting to the API</h2>
+        <p>
+          The Python Library connects to the Linode API V4 using the LinodeClient class, which expects an OAuth Token in his constructor.
```
Comment (eatonphil) [inline docs/src/components/guides/python/Introduction.js]: s/Library/library/g?

**Pull request merged**